### PR TITLE
add copyright and license info

### PR DESCRIPTION
**Pull request by DakkJaniels**

### CODE REVIEW

**Review by unapproachable — COMMENTED** [review]
Quick review.  We should also have Hari sign off on the Copyright since he's named in it.

File: cli/cli.py [hunk 1]
```diff
@@ -271,11 +303,42 @@ def test_notifications(disable_sound):
     time.sleep(5)
 
 
+@click.command()
+@click.option("--w", is_flag=True)
```
[the remark this answers is not in the document]
Comment (DakkJaniels) [reply cli/cli.py:307]: I was just matching the start up text, so we should be good with this as is I think (how many people are actually going to look at this anyway?) 
image

File: cli/cli.py [hunk 2]
```diff
@@ -271,11 +303,42 @@ def test_notifications(disable_sound):
...
+        show_file = "show_w.txt"
+    elif c:
+        show_file = "show_c.txt"
+    else:
+        print(
+            "Option missing, you must include w or c with show argument. Program Quitting"
+        )
+        exit(0)
+
+    if os.path.exists(LICENSE_PATH):
```
[the remark this answers is not in the document]
Comment (DakkJaniels) [reply cli/cli.py:323]: should be less duplicate code now, at least in cli.py. I moved the startup text to the main file, app.py

File: cli/cli.py [hunk 3]
```diff
@@ -7,20 +26,32 @@
 
 import click
```
[the remark this answers is not in the document]
Comment (DakkJaniels) [reply cli/cli.py:27]: hmm... not sure how that got there, looks like it was already there. (looks like you added it on commit: 8c98222535a3b6ff9e20999f098323d62386018d).

But yes, that should be removed.

**Review by Hari-Nagarajan — APPROVED** [review]

**Pull request merged**